tokio-console/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "tokio-console"
-version = "0.1.10"
+version = "0.2.0"
```

There is not really any breaking change in this version of `tokio-console`, it should be `0.1.11`
```suggestion
version = "0.1.11"
```